src/modules/75-cf/pages/feature-flags-detail/targeting-rules-tab/components/disabled-feature-tooltip/__tests__/DisabledFeatureTooltip.test.tsx

```diff
@@ -23,6 +24,7 @@ const renderComponent = (): void => {
 }
 
 describe('DisabledFeatureTooltip', () => {
+  beforeEach(() => jest.clearAllMocks())
```

Comment:
Would it kill you to put in a line break before the `test`?
```suggestion
  beforeEach(() => jest.clearAllMocks())
  
```

Reply:
okaaaaaaaay